### ISSUE TITLE
Bug with training continuation

### ISSUE THREAD

**Issue by Arkkienkeli** [cytomining/DeepProfiler#252]
When I used `--epoch` to continue epoch, I had the following error:

```
Traceback (most recent call last):
  File "deepprofiler/__main__.py", line 197, in <module>
    cli(obj={})
  File "/home/jupyter-nmoshkov@broadinst-04e9f/DeepProfiler_test/dp_tf1_test/lib/python3.7/site-packages/click/core.py", line 829, in __call__
    return ctx.invoke(self.callback, **ctx.params)
  File "/home/jupyter-nmoshkov@broadinst-04e9f/DeepProfiler_test/dp_tf1_test/lib/python3.7/site-packages/click/core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "/home/jupyter-nmoshkov@broadinst-04e9f/DeepProfiler_test/DeepProfiler/deepprofiler/learning/training.py", line 45, in learn_model
    model.train(epoch, metrics, verbose=verbose)
  File "/home/jupyter-nmoshkov@broadinst-04e9f/DeepProfiler_test/DeepProfiler/deepprofiler/learning/model.py", line 80, in train
    validation_freq=freq
  File "/home/jupyter-nmoshkov@broadinst-04e9f/DeepProfiler_test/dp_tf1_test/lib/python3.7/site-packages/comet_ml/monkey_patching.py", line 292, in wrapper
    return_value = original(*args, **kwargs)
  File "/home/jupyter-nmoshkov@broadinst-04e9f/DeepProfiler_test/dp_tf1_test/lib/python3.7/site-packages/keras/legacy/interfaces.py", line 91, in wrapper
    return func(*args, **kwargs)
  File "/home/jupyter-nmoshkov@broadinst-04e9f/DeepProfiler_test/dp_tf1_test/lib/python3.7/site-packages/keras/engine/training.py", line 1658, in fit_generator
    initial_epoch=initial_epoch)
  File "/home/jupyter-nmoshkov@broadinst-04e9f/DeepProfiler_test/dp_tf1_test/lib/python3.7/site-packages/keras/engine/training_generator.py", line 215, in fit_generator
    class_weight=class_weight)
  File "/home/jupyter-nmoshkov@broadinst-04e9f/DeepProfiler_test/dp_tf1_test/lib/python3.7/site-packages/keras/engine/training.py", line 1449, in train_on_batch
    outputs = self.train_function(ins)
  File "/home/jupyter-nmoshkov@broadinst-04e9f/DeepProfiler_test/dp_tf1_test/lib/python3.7/site-packages/keras/backend/tensorflow_backend.py", line 2979, in __call__
    return self._call(inputs)
  File "/home/jupyter-nmoshkov@broadinst-04e9f/DeepProfiler_test/dp_tf1_test/lib/python3.7/site-packages/keras/backend/tensorflow_backend.py", line 2937, in _call
    fetched = self._callable_fn(*array_vals)
  File "/home/jupyter-nmoshkov@broadinst-04e9f/DeepProfiler_test/dp_tf1_test/lib/python3.7/site-packages/tensorflow_core/python/client/session.py", line 1472, in __call__
    run_metadata_ptr)
tensorflow.python.framework.errors_impl.FailedPreconditionError: 2 root error(s) found.
  (0) Failed precondition: Attempting to use uninitialized value SGD/lr
         [[{{node SGD/lr/read}}]]
         [[loss/add_65/_4049]]
  (1) Failed precondition: Attempting to use uninitialized value SGD/lr
         [[{{node SGD/lr/read}}]]
0 successful operations.
0 derived errors ignored.
```
The fix is very simple, I will do PR for it.

**Comment by jccaicedo** [comment]
Thanks for reporting this and for the PR.
Looks good!

**Comment by jccaicedo** [comment]
Fixed here: e8245d1b96db06057dc667dd97969c78787da47c
